### PR TITLE
Added mongo test container

### PR DESCRIPTION
**Pull request by Mingela**

### CODE REVIEW

File: brvs-core/src/main/java/iroha/validation/transactions/storage/impl/mongo/MongoTransactionVerdictStorage.java [hunk 1]
```diff
@@ -27,8 +28,14 @@
   private final MongoCollection<MongoVerdict> collection;
   private final PublishSubject<String> subject = PublishSubject.create();
 
-  public MongoTransactionVerdictStorage() {
-    mongoClient = MongoClients.create();
+  public MongoTransactionVerdictStorage(String mongoHost, Integer mongoPort) {
```
Comment (dolgopolovwork) [inline brvs-core/src/main/java/iroha/validation/transactions/storage/impl/mongo/MongoTransactionVerdictStorage.java]: Why `Integer`?

File: brvs-core/src/main/java/iroha/validation/transactions/storage/impl/mongo/MongoTransactionVerdictStorage.java [hunk 2]
```diff
@@ -27,8 +28,14 @@
...
+    if (Strings.isNullOrEmpty(mongoHost)) {
+      throw new IllegalArgumentException("MongoDB host must not be neither null or empty");
+    }
+    if (mongoPort < 1) {
```
Comment (dolgopolovwork) [inline brvs-core/src/main/java/iroha/validation/transactions/storage/impl/mongo/MongoTransactionVerdictStorage.java]: Do we really need this check? If so, it looks like we also need to add an upper bound check

File: brvs-core/src/main/java/iroha/validation/transactions/storage/impl/mongo/MongoTransactionVerdictStorage.java (hunk 1 above)
Comment (Mingela) [reply brvs-core/src/main/java/iroha/validation/transactions/storage/impl/mongo/MongoTransactionVerdictStorage.java]: Done

File: brvs-core/src/main/java/iroha/validation/transactions/storage/impl/mongo/MongoTransactionVerdictStorage.java (hunk 2 above)
Comment (Mingela) [reply brvs-core/src/main/java/iroha/validation/transactions/storage/impl/mongo/MongoTransactionVerdictStorage.java]: Done

**Pull request merged**